app/models/miq_queue.rb

```diff
@@ -30,6 +30,7 @@ class MiqQueue < ApplicationRecord
 
   PRIORITY_WHICH  = [:max, :high, :normal, :low, :min]
   PRIORITY_DIR    = [:higher, :lower]
+  WHITE_LIST      = MiqQueue.column_names.map(&:to_sym) - [:id]
```

Comment:
Part of me wants this to be a hardcoded list of the column names.  Dynamically building this list feels like it could bite us vs. the infrequent schema changes we make to the miq_queue table.

Maybe COLUMNS_FOR_REQUEUE intead of WHITE_LIST?